services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/StsCredentialsProvider.java

```diff
@@ -127,6 +150,30 @@ public B asyncCredentialUpdateEnabled(Boolean asyncCredentialUpdateEnabled) {
             return (B) this;
         }
 
+        /**
+         * Configure the amount of time, relative to STS token expiration, that a new STS token should be fetched synchronously.
```

Comment:
Can we change this to: Configure the amount of time, relative to STS token expiration, that the cached credentials are considered stale and should not longer be used. All threads will block until the value is updated.

Reply:
Done